lib/openssl/ccm.rb

```diff
@@ -119,6 +119,7 @@ def mac(data, nonce, additional_data)
       @cipher.reset
       @cipher.encrypt
       @cipher.key = @key
+      @cipher.iv = "\x00" * 16
```

Comment:
I can't tell if this is a bug in OpenSSL or not, but it's required to have the tests passing with OpenSSL 3.0 on Ubuntu 22.04 though